fe1-web/parts/lao/organizer/__tests__/RollCallOpened.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { useRoute } from '@react-navigation/core';
+import { act, fireEvent, render } from '@testing-library/react-native';
+import {
+  Hash, Lao, LaoState, PublicKey, Timestamp,
+} from 'model/objects';
+import STRINGS from 'res/strings';
+import keyPair from 'test_data/keypair.json';
+import { requestCloseRollCall as mockRequestCloseRollCall } from 'network/MessageApi';
+import { OpenedLaoStore } from 'store';
+// @ts-ignore
+import { fireScan as fakeQrReaderScan } from 'react-qr-reader';
+
+import RollCallOpened from '../RollCallOpened'; // Don't change the order, otherwise the tests will fail
```

Comment:
Is the import order still a problem ? I thought you fixed it. If yes, you can safely remove this comment.